rsocket/internal/SetupResumeAcceptor.cpp

```diff
@@ -39,6 +39,10 @@ class OneFrameProcessor : public FrameProcessor {
   }
 
   void processFrame(std::unique_ptr<folly::IOBuf> buf) override {
+    if(!transport_) {
```

Comment:
Should we put DCHECK instead? The caller might assume that the frame is processed, while it is silently ignored, is it a problem?